load/resources/cimr.ttl

```diff
@@ -169,6 +169,14 @@ pti:Diagram.type
   skos:example "Worldview", "Substation";
   cims:pragmatics "Possible values: Worldview, Substation".
 
+# Additional mappings of units to QUDT.
+
+cim:UnitSymbol.NOKperMWh skos:exactMatch unit:CCY_NOK-PER-MegaW-HR .
+cim:UnitSymbol.EURperMWh skos:exactMatch unit:CCY_EUR-PER-MegaW-HR .
+cim:UnitSymbol.DKKperEUR skos:exactMatch unit:CCY_DKK-PER-CCY_EUR .
+cim:UnitSymbol.NOKperEUR skos:exactMatch unit:CCY_NOK-PER-CCY_EUR .
+cim:UnitSymbol.SEKperEUR skos:exactMatch unit:CCY_SEK-PER-CCY_EUR .
+cim:UnitSymbol.H skos:exactMatch unit:H .
```

Comment:
[CIM is lowercase](https://cim.ontotext.com/graphdb/resource?uri=https:%2F%2Fcim.ucaiug.io%2Fns%23UnitSymbol.h&role=subject) `h` and QUDT is `HR`.

Reply:
not true this is a [Henry](https://qudt.org/vocab/unit/H),  the SI unit of electric inductance.